homeassistant/components/blebox/.translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Translations have moved to 'translations' instead of '.translations' in the latest version